docs/18-guides/02-security/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: App Security Guide
+description: Secure your Nordcraft app by avoiding frontend secrets and using a backend to handle sensitive API operations safely
+---
+
+# How to Keep Your Nordcraft App Secure
+
+If you're building with **Nordcraft**, it's important to understand that it's a **front-end only** tool. That means everything you add—every bit of logic, config, or data—is visible to anyone using your app.
+
+This guide covers the key things you should know (and avoid) to keep your Nordcraft app safe, especially when working with APIs or anything remotely sensitive.
+
+---
+
+## First Things First: What “Front-End Only” Really Means
+
+Nordcraft runs **entirely in the browser**. There’s no backend, no secret layer, and nothing hidden from users. So when we say front-end only, we mean:
+
+* Users can see **everything** — your UI logic, network requests, and any data you embed
+* There’s **no built-in way** to protect API keys or hide sensitive operations
+* It’s fantastic for rapid development and sharing ideas, but it’s not a secure place to put private stuff
+
+---
+
+## Don’t Store Secrets in Your App
+
+This is the big one. **Never** put secrets—like API keys, tokens, or credentials—directly into your Nordcraft project. If it’s in your project, it’s exposed.
```

Comment:
```suggestion
Because everything in your Nordcraft apps are visible to users in a browser, you should not store secrets or sensitive information in your Nordcraft projects.
```